### PR TITLE
Fixed a load of docblocks

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: lib/Github/Api/CurrentUser.php [hunk 1]
```diff
@@ -104,7 +104,6 @@ public function teams()
      * @param  string $type      role in the repository
      * @param  string $sort      sort by
      * @param  string $direction direction of sort, ask or desc
-
```
Comment (xabbuh) [inline lib/Github/Api/CurrentUser.php]: Why did you remove this blank line and not, for example, in the diff above?

Comment (GrahamCampbell) [inline lib/Github/Api/CurrentUser.php]: The other's are blank lines of comment. This is just a dodgie blank line with nothing on it at all.

Comment (GrahamCampbell) [inline lib/Github/Api/CurrentUser.php]: I was going to send a pull request later to standardise the annotation separation.

I want to keep the PRs atomic.

Comment (stof) [inline lib/Github/Api/CurrentUser.php]: You should add the missing ``*`` instead IMO

Comment (GrahamCampbell) [inline lib/Github/Api/CurrentUser.php]: @stof Should I fix all cases then? There's loads of them...

Comment (stof) [inline lib/Github/Api/CurrentUser.php]: Well, I would fix this dodgie blank line in the expected way.

Then, fixing the separation of params and return values could be done in a separate PR to make it easier for maintainers to review it, or it could be included in the same PR to have everything fixed at once. I don't know what @pilot prefers

Comment (GrahamCampbell) [inline lib/Github/Api/CurrentUser.php]: Ping @pilot. What would you like me to do here?

Comment (pilot) [inline lib/Github/Api/CurrentUser.php]: @GrahamCampbell make all in one PR, just to fix all at once pls

Comment (GrahamCampbell) [inline lib/Github/Api/CurrentUser.php]: Sure, will do.

**Comment by GrahamCampbell** [comment]
Ping @pilot, @stof. Done. :)

File: lib/Github/Api/Repo.php [hunk 2]
```diff
@@ -25,7 +25,8 @@
 class Repo extends AbstractApi
 {
     /**
-     * Search repositories by keyword:
+     * Search repositories by keyword:.
```
Comment (pilot) [inline lib/Github/Api/Repo.php]: remove dot at the end for all places, looks weird

Comment (GrahamCampbell) [inline lib/Github/Api/Repo.php]: This is happening because you're miss using the short description.

Comment (GrahamCampbell) [inline lib/Github/Api/Repo.php]: Also, a dot is required because this is a sentence.

Comment (guillermoandrae) [inline lib/Github/Api/Repo.php]: @GrahamCampbell: in this case, you just need to remove the colon before the period.

Comment (GrahamCampbell) [inline lib/Github/Api/Repo.php]: Sure I'll do a find and replace for all cases.

Comment (GrahamCampbell) [inline lib/Github/Api/Repo.php]: Done.

**Comment by GrahamCampbell** [comment]
Ping @pilot.
